core/lib.rs

```diff
@@ -81,6 +83,11 @@ pub use crate::runtime::JsErrorCreateFn;
 pub use crate::runtime::JsRuntime;
 pub use crate::runtime::RuntimeOptions;
 pub use crate::runtime::Snapshot;
+// pub use crate::runtime_modules::declare_ops!;
+// pub use crate::runtime_modules::include_js_files!;
```

Comment:
Remove?

Reply:
Yeah, I added those so when reviewing "exports" by checking `lib.rs` it's apparent that we export macros.